api/src/main/java/io/minio/MinioClient.java

```diff
@@ -4763,7 +4727,10 @@ private String putObject(String bucketName, String objectName, Object data, int
    *          Size of object data.
    * @param data
    *          Object data.
+   *
+   * @deprecated As of release 7.0
    */
+  @Deprecated
```

Comment:
Do we need to deprecate a private function ?

Reply:
Yep